express/assets/css/app.scss

```diff
@@ -178,3 +178,7 @@ code {
   border-left: 5px solid $govuk-link-colour;
   padding: 24px 24px 0 24px;
 }
+
+main.main-account-exists {
+  padding-top: 15px;
+}
```

Comment:
We shouldn't need custom padding for this page. We should stick to the design system's padding unless we have a custom design (which we don't for this page).